src/components/EpisodesControls/renderUtils.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { s } from "./";
+import Link from "COMPONENTS/Link/";
+
+function renderPaginationLink(data){
+
+	const { number } = data;
+
+	return (
+		<li key={`episode__pagination__${number}`}>
```

Comment:
I've just found out that [aria-current](https://www.w3.org/TR/wai-aria-1.1/#aria-current) exists! can we use that here? If we grab the `activeIndex` from the provider and compare it against the `index` here, then we should be able to apply that.

Reply:
Now importing context and setting aria-current based on activeindex.